lib/widgets-page.php

```diff
@@ -77,7 +77,7 @@ function gutenberg_widgets_init( $hook ) {
 		gutenberg_get_legacy_widget_settings()
 	);
 
-	$settings = gutenberg_experimental_global_styles_settings( $settings );
+	$settings = apply_filters( 'block_editor_settings', $settings, null );
```

Comment:
I don't think we should do that tbh, this will bring a ton of settings and features from the post editor that we're uncertain whether we want or not; Maybe it's good but I'd rather do it setting per setting.

As mentionned, the styles is a good one where I think they are different. The widgets should just use the default editor styles.